openapi.json

```diff
@@ -0,0 +1,263 @@
+{
+    "openapi": "3.1.0",
+    "info": {
+        "title": "Unstructured Pipeline API",
+        "version": "0.0.1",
```

Comment:
is this visible anywhere? should this version mirror the version of the release?

Reply:
This is unused by Speakeasy right now. Once the FastAPI spec is the source of truth, it will be set properly.